### PR TITLE
Cleanup start demo JS

### PR DESCRIPTION
**Pull request by jonathantneal**
Organizes and labels sections of code, and renames variables for higher readability.

### CODE REVIEW

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: node/magnet/src/components/Start.js [hunk 1]
```diff
@@ -115,229 +172,336 @@ export default class Start extends Component {
...
+      return runHtmlThenInput('\n     <b>/yoursite</b>\n');
+    }
+
+    /* runSudoWeDeployThenInput: harlem shake the page
+    /* ====================================================================== */
+
+    function runSudoWeDeployThenInput() {
+      const scriptEl = document.createElement('script');
+
+      scriptEl.src = 'https://rawgit.com/jonathantneal/656b23d080994df1587f770f61d88c77/raw/a083b647b930ab11417c6c820d714632f3524cc0/harlem-shake.js';
```
Comment (zenorocha) [inline node/magnet/src/components/Start.js]: Move script inside the project

Comment (jonathantneal) [reply node/magnet/src/components/Start.js]: Done. Thanks. I haven’t optimized this script, but I did format it to better follow our spacing conventions.

**Comment by jonathantneal** [comment]
@zenorocha, this is ready for review.

**Pull request merged**